src/backend/internal/controller/user/me.go

```diff
@@ -0,0 +1,40 @@
+package user
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Roongkun/software-eng-ii/internal/model"
+	"github.com/gin-gonic/gin"
+)
+
+func (r *Resolver) Me(c *gin.Context) {
```

Comment:
I think this function is almost the same as the `get_user_info` one, but having this could benefits front-end and the readability of the code, hence I am fine with it.

However, please consider change the name of the function, i.e., `getMyUserInfo()`.